Changelog.md

```diff
@@ -8,6 +8,9 @@
 - Replace uses of first name and last name attributes with display name (#5844)
 - Upgrade to Rails 7 (#5885)
 
+## [v2.0.9]
```

Comment:
@anoverga17 Our process is to write the Changelog entry under "unreleased", and then when the release is made, the entry gets moved to a new section.

Reply:
I see. I must have missed this from the wiki. Thanks for letting me know.